### ISSUE TITLE
Bug: XAML Controls Gallery: Accessibility: [MAS4.2.1]: Narrator fails to announce the <count> of Inbox messages when navigated there and Users are unable to navigate to other symbols under 'Different info badge styles' heading.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#5878]
**Tags**
A11yFeature, Accessibility, A11yMAS, A11ySev2, MAS4.2.1, XAML Controls Gallery (Debug), ScreenReader, Win32, Wipro

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to search bar and type Info Badge and hit enter key.
3. Now Tab to navigate to the Home tab and then use Down arrow key to navigate to 'Inbox' and listen to Narrator announcement.
4. Narrator fails to announce the <count> of unread/new mails number which is present over inbox button.
5. Now try to navigate to other different info badge symbols using 'Caps+right/Left' arrow keys and observe only one of the info badge symbol is getting focussed but not the other symbol.

**Actual behavior**
Users are unware of the presence of the info badge symbols over the buttons as narrator fails to announce aby information regarding them.
Also few symbols under 'Different info badge symbols' are unable to focus using 'Caps+arrow' keys as well.

**Expected behavior**
Narrator should announce the information about these info badge symbols whenever navigated and users must be able to navigate to all the info badge symbols present in this page.

**Version Info**
Microsoft OS: 22449.1000
App: XAML Controls Gallery (Debug) (Version 1.2.16.0)

**Attachment**

1. inbox count.png
![inbox count](https://user-images.githubusercontent.com/74585893/133264858-8aa792d0-24ae-4bb5-bf65-ab310bffd257.png)
2. diff info badge.png
![diff info badge](https://user-images.githubusercontent.com/74585893/133265231-cb9ded07-ef16-4f65-839e-7dc0cb338f09.png)

**Comment by StephenLPeters** [comment]
It is the applications responsibility to raise narrator events when the value changes and to be read out with appropriate context when narrator focus's its parent item.  Moving to XCG as a valid app issue.
